plone/app/layout/viewlets/document_byline.pt

```diff
@@ -2,12 +2,6 @@
      id="plone-document-byline"
      i18n:domain="plone"
      tal:condition="view/show">
-  <span id="lock-icon"
-        tal:define="lock_icon view/locked_icon"
-        tal:condition="python:lock_icon">
-      <img tal:replace="structure lock_icon" />
-  </span>
```

Comment:
yes; it has problems, but is available; see https://github.com/plone/Products.CMFPlone/issues/1056